src/tokens/backgroundColor.js

```diff
@@ -23,6 +23,8 @@ const light = {
   LeftNavItemHover: core.Color.Neutral100,
   LeftNavItemActive: core.Color.Primary100,
   TabHover: core.Color.Neutral100,
+  HorizontalRowLight: core.Color.Neutral300,
```

Comment:
Nice try Brian. Hr stands for Horizontal Rule, not Horizontal Row 😜 You will want to change that before you merge it.

Reply:
Changed